source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -449,8 +449,7 @@ FilterTrailersStatus Filter::onTrailers(ProcessorState& state, Http::HeaderMap&
   state.setTrailersAvailable(true);
   state.setTrailers(&trailers);
 
-  if (state.callbackState() == ProcessorState::CallbackState::HeadersCallback ||
-      state.callbackState() == ProcessorState::CallbackState::BufferedBodyCallback) {
+  if (state.callbackState() != ProcessorState::CallbackState::Idle) {
     ENVOY_LOG(trace, "Previous callback still executing -- holding header iteration");
```

Comment:
Is this log accurate any more?  we are holding more than just header iteration?

Reply:
My understanding is that trailer is also headers. I guess that's what the log means.  We can change it into "holding trailer iteration" if that is more precise.